### PR TITLE
Rira 7.adjustment to many services and tests

### PR DESCRIPTION
**Pull request by migowskiD**
Implementation of dependency injection. The idea is to allow creating router with the specific service class. Each class which can be a parameter for router constructor class should be of the same type (should inherit from the same class). For each set of service classes, which are used to create routers there is defined a seperate configuration, which is given as a variable in docker-compose.yml.

### CODE REVIEW

**Review by mrwrob — CHANGES_REQUESTED** [review]
It looks really good :+1: 
All tests in  `tests/tests_*` directories refer now to the GraphDB backend. Therefore, I suggest to create new subdirectory inside `tests/`, lets called id `tests_graphdb`, and move all the `tests_*` directories  there.

Please address all other comments and we can marge it :smile:

File: grisera_api/services.py [hunk 1]
```diff
@@ -59,120 +59,120 @@ def __init__(self):
 
     def activity_service(self) -> ActivityService:
         if self.persistence_type == PersistenceTypes.GRAPHDB:
-            return ActivityServiceGraphDB()
+            return ActivityServiceGraphDB
```
Comment (mrwrob) [inline grisera_api/services.py:62]: Not that way :facepalm: I though that my comment was pretty clear... You should add parentheses, not remove them...
```python
    def activity_service(self) -> ActivityService:
        if self.persistence_type == PersistenceTypes.GRAPHDB:
            return ActivityServiceGraphDB()
        else:
            return ActivityService()
```

**Review by mrwrob — CHANGES_REQUESTED** [review]

**Review by mrwrob — APPROVED** [review]
Good work!

**Review by Kajeszuk — APPROVED** [review]
Great code!

**Pull request merged**